### ISSUE TITLE
Make `meshscatter` and `scatter` the same function?

### ISSUE THREAD

**Issue by c42f** [JuliaPlots/AbstractPlotting.jl#90]
Is there a reason why we have both of these? Their functionality looks almost exactly the same.

Can we collapse them together by just replacing `meshscatter` with `scatter(..., marker=some_mesh)` and dispatching on the `marker` to get the correct geometry processing and shader setup?

**Comment by mkborregaard** [comment]
I agree, we've talked about that on Slack before. Another alternative we talked about would be for the markers of `scatter` to have axis units and `meshscatter` to have model units by default, but on second thought I think that is less clear.

**Comment by c42f** [comment]
Yes, fundamentally both of these are about points: they place an object in the scene at each supplied position. Presentation being defined by the attributes.

Having said that, many of the attributes to `scatter` are billboard or sprite-related and would be irrelevant for other types of geometry. That's not necessarily a problem, but it feels like a point of confusion.
